mappings/net/minecraft/sortme/Hand.mapping

```diff
@@ -1,3 +1,3 @@
-CLASS aie net/minecraft/sortme/OptionMainHand
+CLASS aie net/minecraft/sortme/Hand
```

Comment:
What's this used for? There's already the Hand enum...maybe a different name would be best

Reply:
yeah, having two classes named `Hand` is a bad idea.

Reply:
It's passed as an argument to ModelWithArms.setHandAngle.

Reply:
Since ther's another one, I'm guessing both are called the same by mojang and they used the options one for that method,

Reply:
The one enum represents `LEFT` or `RIGHT`, while the other represents `MAIN` or `OFFHAND`. It does not seem like a mistake to be using `LEFT`/`RIGHT` here.

Reply:
`OptionMainHand` is about which hand is set to be the main hand. The current name is just fine.

Reply:
It doesn't represent how it's used, though. Like here it's used to render actions on either the left/right player arm.

Reply:
hmm, that's a good point. `Hand` clashes though, which is an issue. I can think of maybe calling this one `ChiralHand`, but that's pretty obtuse.

Reply:
How about DominantHand?

Reply:
Wait, never mind. Because it still has both dominant and non dominant hands.

Reply:
Has the same issue as the former name, imo. It describes its use as an option, but not in cases such as `ModelWithArms#setHandAngle`

Reply:
Is it used exclusively client side?

Reply:
Well even if it's not, it's used exclusively for rendering. How about VisibleHand or RenderingHand?

Reply:
I won't suggest any names because I know the MCP mapping, but probably best to consider the name just by the enum variants: current `Hand` is `MAIN`/`OFFHAND` and `OptionsMainHand` is `LEFT/RIGHT`

Reply:
Renamed it to `AbsoluteHand` (since it's not relative to whether the owner is left or right handed). It might not be the best name, but at least it's better than `OptionMainHand`.